lib/matplotlib/gridspec.py

```diff
@@ -18,6 +18,7 @@
 
 import matplotlib as mpl
 from matplotlib import _api, _pylab_helpers, _tight_layout
+from matplotlib._api import UNSET as _UNSET
```

Comment:
We already have `_api` imported above, though I guess it makes the signature too long?

Reply:
Yes, the idea is to make the signature more readable, but no strong preference.